lib/class-wp-rest-menu-items-controller.php

```diff
@@ -524,51 +524,6 @@ protected function prepare_item_for_database( $request ) {
 			}
 		}
 
-		// If menu id is set, valid the value of menu item position and parent id.
-		if ( ! empty( $prepared_nav_item['menu-id'] ) ) {
```

Comment:
Menu id validation is not need here, as this is done upstream.

Reply:
Sorry, can you clarify what you mean by "upstream" here?

Reply:
`menu-id` maps to `menus` when you submit it. Checking it term existings is all handled by [handle_terms](https://github.com/WordPress/wordpress-develop/blob/092ee0fbffaa8723cbddd1134ec811fdcf8601b8/src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php#L1500-L1525). I got the same error. I can return it.

Reply:
Gotcha, that seems fine then to me.